programs/svm-spoke/src/instructions/create_token_accounts.rs

```diff
@@ -0,0 +1,46 @@
+use anchor_lang::prelude::*;
+use anchor_spl::{ associated_token::{ self, AssociatedToken }, token_interface::{ Mint, TokenInterface } };
+
+use crate::error::SvmError;
+
+#[derive(Accounts)]
+pub struct CreateTokenAccounts<'info> {
+    #[account(mut)]
+    pub signer: Signer<'info>,
+
+    #[account(mint::token_program = token_program)]
+    pub mint: InterfaceAccount<'info, Mint>,
+
+    pub token_program: Interface<'info, TokenInterface>,
+
+    pub associated_token_program: Program<'info, AssociatedToken>,
+
+    pub system_program: Program<'info, System>,
+}
+
+pub fn create_token_accounts<'info>(ctx: Context<'_, '_, '_, 'info, CreateTokenAccounts<'info>>) -> Result<()> {
```

Comment:
why in some cases do we need this context syntax like this `'_, '_, '_,`? we do the same thing in `handle_receive_message`. I'm not really clear on this.

Reply:
Context has 4 lifetime parameters. and since we need `'info` when referencing remaining accounts in CPI we need to pass `_` placeholders for all others